packages/gatsby-theme-carbon/src/styles/internal/_page.scss

```diff
@@ -166,3 +166,21 @@ img {
 .bx--inline-notification--low-contrast:before {
   pointer-events: none !important;
 }
+
+//---------------------------------------
+// Not-nested bullet alignment
+//---------------------------------------
+
+.bx--list--unordered,
+.bx--list--ordered {
+  margin-left: 1rem;
+}
+
+//---------------------------------------
+// Nested bullet alignment
+//---------------------------------------
+
+.bx--list--nested .bx--list--unordered,
+.bx--list--ordered {
```

Comment:
I don't think these selectors are doing anything. When I inspect it, the margin is still 1rem. 
`bx--list--unordered` is the same as `bx--list--nested`, as in they're both the `ul` or `ol`. Seems like it should either be:

```css
.bx--list--unordered.bx--list--nested, 
.bx--list--ordered.bx--list--nested { 
...
}
```

 or

```css
.bx--list--nested .bx--list__item {
...
}
```
if you're trying to target the item inside.